### PR TITLE
Restructuring readme

### PR DESCRIPTION
**Pull request by benlower**
Simplifying some the content. Putting features up top.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,33 +4,36 @@
...
 
-## What is AI.JSX?
+## About AI.JSX
 
-AI.JSX is a framework for building AI applications using Javascript and [JSX](https://react.dev/learn/writing-markup-with-jsx). While AI.JSX [is not React](https://docs.ai-jsx.com/is-it-react), it's designed to look and feel very similar while also integrating seamlessly with React-based projects. With AI.JSX, you don't just use JSX to describe what your UI should look like, you also use it to describe how **Large Language Models**, such as ChatGPT, should integrate into the rest of your application. The end result is a powerful combination where _intelligence_ can be deeply embedded into the application stack.
+AI.JSX is a framework for building AI applications using Javascript and JSX. With AI.JSX, you can easily orchestrate multiple Large Language Model (LLM) calls expressed as modular, reusable components, and interweave UI components with your AI components. This means you can rely on the LLM to construct your UI dynamically from a set of provided React components. AI.JSX also provides native support for tools, Document Question & Answering, and much more.
```
Comment (zkoch) [inline readme.md:9]: I actually think we we can simplify this paragraph even more.

Instead of "orchestrate multiple LLM calls expressed as..." let's just call it what it is: great tools for prompt engineering. Instead of "interweave UI components with AI components", let's just say you can have the LLM render React components in its response instead of just text.

Basically, I think we should evolve this to the eli5 version

Comment (NickHeiner) [reply readme.md]: 💯 for the "above the fold README" being ELI5.

Comment (benlower) [reply readme.md]: just committed an update to simplify per Zach's feedback...thoughts?

File: readme.md [hunk 2]
```diff
@@ -4,33 +4,36 @@
...
-1. The ability to seamlessly interweave UI components with your AI components. This means you can rely on the LLM to construct your UI dynamically from a set of provided React components.
+## Features
 
-AI.JSX can be used to create standalone LLM applications that can be deployed anywhere Node.JS is supported, or it can be used as part of a larger React application. For an example of how to integrate AI.JSX into a React project, see the [NextJS demo package](/packages/nextjs-demo/) or [follow the tutorial](https://docs.ai-jsx.com/category/tutorial). For an overview of all deployment architectures, see the [architecture overview](https://docs.ai-jsx.com/guides/architecture).
+AI.JSX comes with the following features out-of-the-box:
 
-For more details on how AI.JSX works with React in general, see our [AI+UI guide](https://docs.ai-jsx.com/guides/ai-ui).
+- **Componetized** → LLM prompt engineering through modular, reusable components.
+- **Model Support** → Use Llama2 and models from Anthropic and OpenAI. Seamlessly switch between model providers and LLM config (e.g. temperature).
```
Comment (NickHeiner) [inline readme.md:18]:
```suggestion
- **Model Support** → Use OpenAI, Antrhopic, and Llama2, or bring your own model. Seamlessly switch between model providers and LLM config (e.g. temperature).
```

File: readme.md [hunk 3]
```diff
@@ -4,33 +4,36 @@
...
+- **Complete AI Toolbox** → Built-in support for Tools (ReAct pattern), Document Question and Answering, Chain of Thought, and more.
+- **Generative UI** → Seamlessly interweave LLM calls with standard UI components. LLM can dynamically render UI from a set of provided components.
```
Comment (NickHeiner) [inline readme.md:20]:
```suggestion
- **Generative UI** → Seamlessly interweave LLM calls with standard UI components. LLM can dynamically render UI from a set of components you provide.
```

File: readme.md [hunk 4]
```diff
@@ -4,33 +4,36 @@
...
+## Features
 
-AI.JSX can be used to create standalone LLM applications that can be deployed anywhere Node.JS is supported, or it can be used as part of a larger React application. For an example of how to integrate AI.JSX into a React project, see the [NextJS demo package](/packages/nextjs-demo/) or [follow the tutorial](https://docs.ai-jsx.com/category/tutorial). For an overview of all deployment architectures, see the [architecture overview](https://docs.ai-jsx.com/guides/architecture).
+AI.JSX comes with the following features out-of-the-box:
 
-For more details on how AI.JSX works with React in general, see our [AI+UI guide](https://docs.ai-jsx.com/guides/ai-ui).
+- **Componetized** → LLM prompt engineering through modular, reusable components.
+- **Model Support** → Use Llama2 and models from Anthropic and OpenAI. Seamlessly switch between model providers and LLM config (e.g. temperature).
+- **Complete AI Toolbox** → Built-in support for Tools (ReAct pattern), Document Question and Answering, Chain of Thought, and more.
```
Comment (NickHeiner) [inline readme.md:19]: I don't think we actually use the `ReAct` pattern.

**Review by NickHeiner — APPROVED** [review]

File: readme.md (hunk 4 above)
Comment (NickHeiner) [reply readme.md:19]: Or CoT

Comment (benlower) [reply readme.md]: removing these
